Source/Core/EllipseGeometry.js

```diff
@@ -980,6 +975,9 @@ define([
          */
         rectangle : {
             get : function() {
+                if (!defined(this._rectangle)) {
+                    this._rectangle = computeRectangle(this);
```

Comment:
It can be in a separate PR, but we should also update `polygon`, `corridor` and `rectangle` to work this way